### ISSUE TITLE
Wallpaper is not set in kde

### ISSUE THREAD

**Issue by charliefrance** [EmperorEntropy/PaletteSnap#2]
When palsnap is run, palette is generated but I get the error
`Cannot find 'org.kde.plasma.desktop.Background' in object /Desktop/Background at org.kde.plasma.desktop`

**Comment by EmperorEntropy** [comment]
Thank you for trying out PaletteSnap! I wasn't confident about the wallpaper setting code for KDE, so the feedback is much appreciated. I don't have Linux myself, so I'll have to rely on people like you to test for me.

Suppose `<image_path>` is the path for your image. Does running the following in the command line set the wallpaper for you on all desktop displays?
```
qdbus org.kde.plasmashell /PlasmaShell org.kde.PlasmaShell.evaluateScript 'var allDesktops = desktops();print (allDesktops);for (i=0;i<allDesktops.length;i++) {d = allDesktops[i];d.wallpaperPlugin = "org.kde.image";d.currentConfigGroup = Array("Wallpaper", "org.kde.image", "General");d.writeConfig("Image", "file://<img_path>")}'
```

If so, I'll push an update to PaletteSnap to fix this.

**Comment by charliefrance** [comment]
It gives this output, but it does succesfully change my wallpaper.
`WorkspaceScripting::Containment(0x5572ba1d8bc0),WorkspaceScripting::Containment(0x5572bb1de6b0)
`

Great project by the way!

**Comment by EmperorEntropy** [comment]
Thanks for confirming it works! I just released 1.1.0, so be sure to update your version of PaletteSnap. The update should resolve this bug, and you shouldn't see the confirmation message in the terminal. A changelog if you are curious:
- KDE wallpaper setting should work.
- Added HSL for templating.
- Added iterations option for palette generation.
- Made improvements to backup optimization.

Wiki pages have also been updated to account for these changes.

If there are still any issues, please reopen this issue since there is a small chance that I might have messed up the escaping.
